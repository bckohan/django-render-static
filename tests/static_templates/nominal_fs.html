<!DOCTYPE html>
<html>
<body>

<h1>Nominal FS</h1>

<p>Settings secret: {{ settings.SECRET_KEY }}</p>

</body>
</html>
