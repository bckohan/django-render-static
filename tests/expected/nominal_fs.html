<!DOCTYPE html>
<html>
<body>

<h1>Nominal FS</h1>

<p>Settings secret: psst</p>

</body>
</html>
